crates/red_knot_python_semantic/resources/mdtest/import/star.md

```diff
@@ -916,6 +916,20 @@ from a import *, _Y as fooo  # error: [invalid-syntax]
 from a import *, *, _Y  # error: [invalid-syntax]
 ```
 
+### Cyclic star imports
+
+`a.py`:
+
+```py
+from b import *
```

Comment:
I think it would be worth actually defining some real name in one of these modules, and then having a third module `import *` from one of these, and assert that the real name shows up there.